overlay/etc/init.d/etcd-service

```diff
@@ -0,0 +1,28 @@
+#!/sbin/openrc-run
+
+depend() {
+    after network-online
+    want cgroups
+}
+
+start_pre() {
+    mkdir -p /var/lib/etcd-data/
+}
+
+if [ -f /etc/etcd-service.conf ]; then source /etc/etcd-service.conf; fi
```

Comment:
Is this necessary given that openrc will source `/etc/conf.d/etcd-service` if it exists?

Reply:
I kept the arguments written out separate but can move this over to /etc/conf.d/etcd-service. I'll make the change and update the PR.

Reply:
PR Updated so that /etc/conf.d/etcd-service.conf is updated with the arguments from cloud config rather than the separate file